stories/packages/BooleanQueryTokenizer.tsx

```diff
@@ -0,0 +1,55 @@
+import * as React from 'react';
+import { storiesOf } from '@storybook/react';
+import { BooleanQueryTokenizer, TextArea } from '@textkernel/oneui';
+
+const tokenizer = new BooleanQueryTokenizer();
+const initialValue =
+    '-php AND +c++ +--+ -\n' +
+    'java* NOT *script\n' +
+    'NEAR (rust OR tex*nel) OR\n' +
+    '-.NET "TO BE OR NOT TO BE"';
+
+const columnStyle = {
+    width: '30%',
+    padding: '1em',
+};
+
+storiesOf('packages|BooleanQueryTokenizer', module).add('BooleanQueryTokenizer', () => {
```

Comment:
Good call creating a section for packages. Maybe can move the highlighter here as well?

Reply:
Since Highlighter is a mix of components and package, I guess it shouldn't be used as a package only.
And current Highlighter story is made for component, so it would be weird to move it under package directory.

Reply:
you are right